server/src/main/java/org/opensearch/tasks/consumer/package-info.java

```diff
@@ -0,0 +1,17 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+/**
+ * Support for adding consumers to consume task related information.
+ */
+/*
+ * Modifications Copyright OpenSearch Contributors. See
+ * GitHub history for details.
+ */
```

Comment:
Do you need this?

Reply:
Dont think so, I included it as Task folder's package-info has the above line. Removed it as other files dont have it.